proxy/http/HttpConnectionCount.h

```diff
@@ -420,9 +420,9 @@ OutboundConnTrack::Linkage::equal(key_type lhs, key_type rhs)
 
 Action *register_ShowConnectionCount(Continuation *, HTTPHdr *);
 
-namespace ts
+namespace swoc
```

Comment:
Seems odd to modify the swoc ns here.  Shouldn't bwformat resolve with ADL rather than adding overloads to swoc?

Reply:
I haven't had much luck with that. I've tried to avoid using `std` but frequently it doesn't compile due to not found or ambiguity. I think one issue is if one is an overload and the other a templated overload, because the first "matches" it doesn't check the better match with the template.